boards/common/stm32/stm32_leds.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2022 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_stm32
+ * @brief       Common LED macros
+ * @{
+ *
+ * @file
+ * @brief       Common LED macros
+ *
+ * @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * This idea is that STM32 boards only define the pin and port number of LEDs
+ * and this header provides the rest of the defines
+ */
+
+#ifndef STM32_LEDS_H
+#define STM32_LEDS_H
+
+/* GPIO_PORT() macro. This define even works when GPIO LL is not in used */
+#include "gpio_ll_arch.h"
```

Comment:
GPIO LL is actually not provided for STM32F1, but if I am not mistaken the vendor header files are compatible enough to just work anyway for th e `GPIO_PORT()` macro, which is all that is used here. It may be worth to actually test this for one STM32F1 board. I will do so once everything compiles.

Reply:
Tested on a bluepill, see https://github.com/RIOT-OS/RIOT/pull/18415#issuecomment-1210772190

So it does in fact work, as I expected.